homeassistant/components/airzone/const.py

```diff
@@ -0,0 +1,43 @@
+"""Constants for the Airzone integration."""
+
+from typing import Final
+
+from aioairzone.const import AZD_HUMIDITY, AZD_TEMP
+
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    SensorEntityDescription,
+)
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    ENTITY_CATEGORY_DIAGNOSTIC,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+)
+
+DOMAIN: Final = "airzone"
+MANUFACTURER: Final = "Airzone"
+
+AIOAIRZONE_DEVICE_TIMEOUT_SEC: Final = 10
+DEFAULT_LOCAL_API_HOST: Final = ""
+DEFAULT_LOCAL_API_PORT: Final = 3000
+
+SENSOR_TYPES: Final[tuple[SensorEntityDescription, ...]] = (
+    SensorEntityDescription(
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
```suggestion
```

Since these are core functionality of the integration, we probably don't want to set `ENTITY_CATEGORY_DIAGNOSTIC` unless you plan on adding additional platforms later that will make these less relevant

Reply:
Done in 9a96604fc814869ecb594340ace7a1325230b05b